File: Genogram_API/Genogram.Domain/Models/Services/User/UserModel.cs
using Genogram.Domain.Entites;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Genogram.Domain.Models.Services.User
{
    public class UserModel
    {
        public int Id { get; set; }
        public string Name { get; set; } = null!;
        public string? Street { get; set; }
        public string? City { get; set; }
        public string? Pincode { get; set; }
        public string? Nationality { get; set; }
        public string? Language { get; set; }
        public string? Email { get; set; }
        public string? DateOfBirth { get; set; }
        public string? UserImage {  get; set; }
        // Navigation property for related UserRelationEntities
        public ICollection<UserRelationModel> UserRelations { get; set; } = new List<UserRelationModel>();
    }
}

File: Genogram.client/src/app/core/models/UserModel.ts
  export  interface UserModel
        {
            id:number,
            name:string,
            street:string,
            city:string,
            pincode:string,
            nationality:string,
            language:string,
            email:string,
            dateOfBirth:string,
            userImage:string,
            userRelations?:UserRelationalModel[]
        }
export interface UserRelationalModel{
  id:Number,
  firstName:string,
  lastName:string,
  relationship:string,
  phone:string,
  email:string,
  primaryContact:boolean,
  userId:number
}